conf/modules.config

```
@@ -94,6 +94,33 @@ process {
         ]
     }
 
+    withName: 'DRAGONFLYE' {
+        ext.args = {
+            if ( $meta.gsize && $meta.gsize != 'NA' ){
+                if ( !params.dragonflye_args.contains("--gsize") ) {
+                    "--gsize ${meta.gsize} ${params.dragonflye_args}"
+                } else {
+                    params.dragonflye_args ? params.dragonflye_args : ''
```

I think that could be shortened to
```suggestion
                    params.dragonflye_args ?: ''
```
same below once more